lib/generators/sorcery/install_generator.rb

```diff
@@ -90,6 +90,12 @@ def only_submodules?
         options[:migrations] || options[:only_submodules]
       end
 
+      def rails_version # for Rails5 ActiveRecord::Migration 
```

Comment:
This method is called `rails_version`, but it does not return a rails version. It returns a suffix to migration class name that depends on a rails version.

What I suggest is to add a method called `migration_class_name` which would replace the whole `ActiveRecord::Migration<%=rails_version%>` code in the migration generators, and this method could look like this:
```
if Rails::VERSION::MAJOR >= 5
  "ActiveRecord::Migration[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
else
  "ActiveRecord::Migration"
end
```